chrome/inject.browserify.js

```diff
@@ -80,6 +80,10 @@
   update(field(PASSWORD_FIELDS), login.p);
   update(field(USERNAME_FIELDS), login.u);
 
+  if (login.digits !== "") {
```

Comment:
maybe let's use a non-strict `if (login.digits)`? don't want to deal with bugs in future if at some point absence of OTP makes `login.digits` equal to `undefined` or `null` instead of an empty string :)